demo/deploy_demo.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import argparse
+import os
+
+import torch
+from mmengine import ProgressBar
+
+from mmyolo.utils.misc import get_file_list
+
+try:
+    from mmdeploy.apis.utils import build_task_processor
+    from mmdeploy.utils import get_input_shape, load_config
+except ImportError:
+    raise ImportError(
+        'mmdeploy is not installed, please see '
+        'https://mmdeploy.readthedocs.io/en/latest/01-how-to-build/build_from_source.html'  # noqa
+    )
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='For mmdeploy predict')
+    parser.add_argument('img', help='image path')
+    parser.add_argument('--model-cfg', help='model config root')
+    parser.add_argument('--backend-model', help='backend model path')
+    parser.add_argument('--deploy-cfg', help='deploy config path')
+    parser.add_argument(
+        '--out-dir', default='./output', help='Path to output file')
+    parser.add_argument(
```

Comment:
parser.add_argument(
        '--show', action='store_true', help='Show the detection results')